include/ct_utils.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2015-2016 Cray Inc.  All rights reserved.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
+ * BSD license below:
+ *
+ *     Redistribution and use in source and binary forms, with or
+ *     without modification, are permitted provided that the following
+ *     conditions are met:
+ *
+ *      - Redistributions of source code must retain the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer.
+ *
+ *      - Redistributions in binary form must reproduce the above
+ *        copyright notice, this list of conditions and the following
+ *        disclaimer in the documentation and/or other materials
+ *        provided with the distribution.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AWV
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+ * BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+ * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#ifndef CT_UTILS_H
+#define CT_UTILS_H
+
+#ifndef CT_FIVERSION
+#define CT_FIVERSION FI_VERSION(1,3)
+#endif
+
+static inline void ct_print_fi_error(const char *fi_fname, int ret_val)
+{
+	fprintf(stderr, "%s() ret=%d (%s)\n", fi_fname, ret_val,
+		fi_strerror(ret_val));
+	fflush(stdout);
+}
+
+static inline void ct_print_opts_usage(const char *opt, const char *desc)
+{
+	fprintf(stderr, " %-20s %s\n", opt, desc);
+}
+
+void ct_parseinfo(int op, char *optarg, struct fi_info *hints);
+
+/* general utilities */
+#include <sys/time.h>
+static inline uint64_t get_time_usec(void)
+{
+	struct timeval tv;
+	uint64_t usecs;
+
+	gettimeofday(&tv, NULL);
+	usecs = (tv.tv_sec * 1000000) + tv.tv_usec;
+	return usecs;
+}
+
+/* job launch stuff */
+#include "pmi.h"
+
+void CT_Init(int *, char ***);
```

Comment:
Good idea.  This was the same way they were in the fabtests-cray version probably for distinguishing our utils, but there's no need for that anymore.  Actually, this particular one probably should be `ct_utils_init`.